tomee/tomee-microprofile/mp-common/pom.xml

```diff
@@ -27,6 +27,128 @@
+        <artifactId>microprofile-fault-tolerance-api</artifactId>
+        <version>${microprofile.fault-tolerance.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.health</groupId>
+        <artifactId>microprofile-health-api</artifactId>
+        <version>${microprofile.health.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.jwt</groupId>
+        <artifactId>microprofile-jwt-auth-api</artifactId>
+        <version>${microprofile.jwt.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.metrics</groupId>
+        <artifactId>microprofile-metrics-api</artifactId>
+        <version>${microprofile.metrics.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.openapi</groupId>
+        <artifactId>microprofile-openapi-api</artifactId>
+        <version>${microprofile.openapi.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.opentracing</groupId>
+        <artifactId>microprofile-opentracing-api</artifactId>
+        <version>${microprofile.opentracing.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.rest.client</groupId>
+        <artifactId>microprofile-rest-client-api</artifactId>
+        <version>${microprofile.rest-client.version}</version>
+      </dependency>
+      <!-- SmallRye API & Impl -->
+      <dependency>
+        <groupId>io.smallrye.config</groupId>
+        <artifactId>smallrye-config</artifactId>
+        <version>${microprofile.config.impl.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-fault-tolerance-api</artifactId>
+        <version>${microprofile.fault-tolerance.impl.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-fault-tolerance-autoconfig-core</artifactId>
+        <version>${microprofile.fault-tolerance.impl.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-fault-tolerance-core</artifactId>
+        <version>${microprofile.fault-tolerance.impl.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-health-api</artifactId>
+        <version>${microprofile.health.impl.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-health</artifactId>
+        <version>${microprofile.health.impl.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-jwt</artifactId>
+        <version>4.0.0</version>
```

Comment:
any reason for not having a property here?

Reply:
it seems that TomEE does not use SmallRye Impl for JWT.
however this could be useful for outside users that need a BOM for MP 5.0 with all impls.

i'll rework a property in the main POM.